src/containers/TrendContainer.js

```diff
@@ -96,11 +98,26 @@ TrendContainer.propTypes = {
   until: PropTypes.number.isRequired,
 }
 
-const mapStateToProps = ({ filters, summaries }) => ({
-  ...filters,
-  ...getPlaceInfo(filters),
-  summaries,
-})
+export const mapStateToProps = ({ filters, summaries }) => {
```

Comment:
Hmm, that's an interesting approach. I do like pulling data out of the summaries object but I see how the way I have done this is more involved than required and it might just be better to pass in a reliable places prop as a way of limiting that access. Let me give it a shot